rapid7_insightvm_cloud/icon_rapid7_insightvm_cloud/actions/stop_scan/action.py

```diff
@@ -0,0 +1,28 @@
+import insightconnect_plugin_runtime
+from .schema import StopScanInput, StopScanOutput, Input, Output, Component
+# Custom imports below
+import requests
+
+
+class StopScan(insightconnect_plugin_runtime.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="stop_scan",
+            description="Stop a currently running scan",
+            input=StopScanInput(),
+            output=StopScanOutput(),
+        )
+
+    def run(self, params={}):
+        scan_id = params.get(Input.ID)
+        url = "scan/"+scan_id+"/stop"
```

Comment:
Personally a fan of f-strings: https://realpython.com/python-f-strings/

So something like:
`f"scan/{scan_id}/stop"`